src/font.js

```diff
@@ -52,7 +52,8 @@ function createDefaultNamesInfo(options) {
  * @property {Number} ascender
  * @property {Number} descender
  * @property {Number} createdTimestamp
- * @property {string=} weightClass
+ * @property {Number} weightClass
+ * @property {Number} italicAngle
```

Comment:
Oh and I think the slope property is missing here

Reply:
I looked through the code (which I picked up from the other pull request) `slope` was being added as a property to the font object, and also added to the `hhea` table... but there is no `slope` property on the `hhea` table, and the font property was not being used for any calculation. So, I removed `slope` from the PR.

Reply:
But it's a property of the font object, so it should be added here. Or am I missing something?

Reply:
Oh, now I get what you're saying. So the slope shouldn't have been part of this PR in the first place, got you!

Reply:
Yeah, I think Lmpessoa added it, thinking he would use it to also work on the `hhea` table values `caretslopeRise` and `caretSlopeRun`... but he decided not to do that part as it wasn't necessary for fonts being recognized as a family. `slope` was declared but never used anywhere.